### PR TITLE
IFTAQCAB-61 Verify functionality to delete test and add Gson lib

### PR DESCRIPTION
**Pull request by Obitotski**
Added changes (plugin) to pom.xml (about encoding UTF-8 - forgot to add it previously)
Added Gson lib
Added changes to the API method (impl Gson)
Added test for deleting test

### CODE REVIEW

File: src/main/java/com/softserveinc/ita/steps/TestsSteps.java [hunk 1]
```diff
@@ -3,20 +3,23 @@
...
+import com.softserveinc.ita.pageobjects.modals.DeletingFormModal;
 
-import static com.softserveinc.ita.util.DataProvider.*;
+import static com.softserveinc.ita.util.DataProvider.ADMIN_LOGIN;
+import static com.softserveinc.ita.util.DataProvider.ADMIN_PASSWORD;
 
 public class TestsSteps {
+    private TestsPage page;
 
     public TestsPage openTestsPage(String subject) {
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/steps/TestsSteps.java:14]: stop returning anything by methods in step classes, init page variables here, but try to perform all operations in step class if needed, asserts also could be performed here

**Review by Nikodych — CHANGES_REQUESTED** [review]

**Comment by Obitotski** [comment]
Added test for JSON schema
Added confs to test runner

File: src/test/java/com/softserveinc/ita/TestsTest.java [hunk 2]
```diff
@@ -111,6 +117,12 @@ public void verifyDeletingTest() {
         soft.assertAll();
     }
 
+    @Test(groups = "positive")
+    @Description("Test to verify corectness of the json schema of tests entities")
+    public void verifyTestEntitySchemaRecords () {
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/TestsTest.java:122]: better to create a separate class for schema verifications, and without using of testrunner

**Review by Nikodych — CHANGES_REQUESTED** [review]

File: src/test/java/com/softserveinc/ita/ApiJsonSchemaTest.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import static com.softserveinc.ita.util.AuthApiUtil.authAsAdmin;
+import static com.softserveinc.ita.util.AuthApiUtil.getSessionsCookie;
+import static com.softserveinc.ita.util.DataProvider.API_LOGOUT_PATH;
+
+public class ApiJsonSchemaTest {
+
+    private Cookie sessionId;
+
+    @BeforeClass(groups = {"positive", "negative"})
+    public void setUpTestsTests() {
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/ApiJsonSchemaTest.java:20]: rename

**Review by Nikodych — CHANGES_REQUESTED** [review]

**Review by Nikodych — APPROVED** [review]

**Pull request merged**